.github/CODEOWNERS

```diff
@@ -107,6 +107,7 @@ extension/observer/ecsobserver/                                     @open-teleme
 extension/observer/ecstaskobserver/                                 @open-telemetry/collector-contrib-approvers @rmfitzpatrick
 extension/observer/hostobserver/                                    @open-telemetry/collector-contrib-approvers @MovieStoreGuy
 extension/observer/k8sobserver/                                     @open-telemetry/collector-contrib-approvers @rmfitzpatrick @dmitryax
+extension/observer/cfgardenobserver/                                @open-telemetry/collector-contrib-approvers @cemdk @tomasmota @m1rp @jriguera 
```

Comment:
Nit: Could this line be moved to be in alphabetical order? (I assume this was added manually, but let me know if this was generated automatically and placed out of order)

Also, I need to be added to this list as well.